tests/templates/pr-create.md.ejs.js

```diff
@@ -7,6 +7,8 @@
 
 "use strict";
 
+/* global describe, it */
```

Comment:
Thanks. I'll copy it to `tests/templates` since the rest of the folders don't need it.